patches/storybook-addon-pseudo-states+3.1.1.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/storybook-addon-pseudo-states/dist/preview.js b/node_modules/storybook-addon-pseudo-states/dist/preview.js
+index 0f8e4e2..a634b18 100644
+--- a/node_modules/storybook-addon-pseudo-states/dist/preview.js
++++ b/node_modules/storybook-addon-pseudo-states/dist/preview.js
+@@ -2,8 +2,9 @@
+ 
+ var coreEvents = require('@storybook/core-events');
+ var previewApi = require('@storybook/preview-api');
++var addons = require('@storybook/addons');
```

Comment:
addons from these 2 places result in different channels.
1. `import { addons } from "@storybook/preview-api"`
2. `import { addons } from "@storybook/addons"`

1) is for Storybook 7+ and 2) Storybook 6 and below. For some reason, `"@storybook/preview-api"` is available for storybook 6 and below, but it just resolves in a different channel, so when the addon calls `addons.getChannel().on(EVENT)` they will not be triggered.

The result of this patch, is for the function [rewriteStyleSheets](https://github.com/chromaui/storybook-addon-pseudo-states/blob/452bf772cbd04dcf4dbd3bf84730812bb3230f79/src/preview/withPseudoState.ts#L175) to be called which adds pseudo hover style to all `:hover` elements to simulate hover styles.